src/components/Molecules/ShareButton/ShareButton.js

```diff
@@ -15,6 +16,8 @@ import {
 const handleShare = (e, typeOfShare, urlToShare) => {
   e.preventDefault();
 
+  ShareTracking(typeOfShare.toLowerCase());
```

Comment:
I'd probably do the lowercasing in the `ShareTracking` function. That way we don't have to just _know_ that this function wants a lowercased value.

Reply:
Sorry another minor point - would it be more conventional to lowercase the first letter in `ShareTracking` and `ShareUrlHelper`? Def not an expert on JS naming conventions but thought capital first letter implied a class (or React component) rather than a function?

Reply:
Left as the other were in capital but I will change them to correct convention 👍